docs/admin/resource-quota.md

```diff
@@ -48,7 +71,24 @@ Resource Quota is enforced in a particular namespace when there is a
 `ResourceQuota` object in that namespace.  There should be at most one
 `ResourceQuota` object in a namespace.
 
-See [ResourceQuota design doc](../design/admission_control_resource_quota.md) for more information.
+## Compute Resource Quota
+
+The total sum of [compute resources](../user-guide/compute-resources.md) requested by pods
+in a namespace can be limited.  The following compute resource types are supported:
+
+| ResourceName | Description |
+| ------------ | ----------- |
+| cpu | Total cpu limits of containers |
+| memory | Total memory limits of containers
+| `example.com/customresource` | Total of `resources.limits."example.com/customresource"` of containers |
+
+For example, `cpu` quota sums up the `resources.limits.cpu` fields of every
+container of every pod in the namespace, and enforces a maximum on that sum.
+
+Any resource that is not part of core Kubernetes must follow the resource naming convention prescribed by Kubernetes.
+
+This means the resource must have a fully-qualified name (i.e. mycompany.org/shinynewresource)
```

Comment:
Deleted this line cause it turns out that arbitrary resources are not supported by current code.